sdk/src/index.js

```
@@ -1,269 +1,777 @@
+          }
+          return { output: null };
+        }
+        
+      case 'if-else':
+        const condition = inputs.condition !== undefined ? inputs.condition : inputs.input;
+        const trueValue = node.data?.trueValue || inputs.trueValue || condition;
+        const falseValue = node.data?.falseValue || inputs.falseValue || null;
+        
+        // Evaluate condition
+        let result;
+        if (node.data?.expression) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)